LocalApp/LocalApplication.java

```diff
@@ -282,10 +279,11 @@ private static void runManagerInstance() {
             }
         }
         else { // Manager doesn't exist.
-            RunInstancesRequest runManagerRequest = new RunInstancesRequest("ami-b66ed3de", 1, 1);
+
+			RunInstancesRequest runManagerRequest = new RunInstancesRequest(Util.Const.ami, 1, 1);
             runManagerRequest.setInstanceType(InstanceType.T2Micro.toString());
-            //runManagerRequest.setUserData(fileToBase64String("C:\\Users\\Haymi\\Documents\\BGU\\DSP\\bootstart.sh"));
-            runManagerRequest.setKeyName("nirhaymi");
+			runManagerRequest.setUserData(Util.Util.Base64BootstartBuilder("Manager", "", "Manager.Manager"));
```

Comment:
no more bootstart.sh; now creating the user script inside the java (inside Util.Util)